packages/aws-appsync/src/client.ts

```diff
@@ -178,6 +178,7 @@ class AWSAppSyncClient<TCacheShape extends NormalizedCacheObject> extends Apollo
         disableOffline = false,
         offlineConfig: {
             storage = undefined,
+            keyPrefix = undefined,
```

Comment:
Maybe I will set default as an empty string, just in case someone tries `undefined` and then `""`

Reply:
Your comment got me thinking, instead of setting a default of `''`, I am setting any *falsey* value explicitly to the redux-persist default.